converter/src/main/java/gov/cms/qpp/conversion/model/Node.java

```diff
@@ -315,16 +309,23 @@ public void setDefaultNsUri(String newDefaultNsUri) {
 	 */
 	private List<Node> findNode(TemplateId templateId, Predicate<List<Node>> bail) {
 		List<Node> foundNodes = new ArrayList<>();
-		if (this.type == templateId) {
-			foundNodes.add(this);
-		}
-		for (Node childNode : childNodes) {
+		List<Node> toSearch = Lists.newArrayList(childNodes);
+		Consumer<Node> templateCheck = (node) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "node" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)